target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-lbr20.dts

```diff
@@ -0,0 +1,488 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Netgear LBR20";
+	compatible = "netgear,lbr20";
+
+	chosen {
+		bootargs-append = "ubi.mtd=ubi root=/dev/ubiblock0_0";
+	};
+
+	aliases {
+		led-boot = &led_green_power;
+		led-failsafe = &led_green_power;
+		led-running = &led_green_power;
+		led-upgrade = &led_red_power;
+		label-mac-device = &gmac;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&tlmm 49 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_green_power: led-0 {
+			label = "green:power";
+			function = LED_FUNCTION_STATUS;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&tlmm 22 GPIO_ACTIVE_LOW>;
+		};
+
+		led_red_power: led-1 {
+			label = "red:power";
+			function = LED_FUNCTION_STATUS;
+			color = <LED_COLOR_ID_RED>;
+			gpios = <&tlmm 23 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		lte_rst {
+			gpio-export,name = "lte_rst";
+			gpio-export,output = <1>;
+			gpios = <&tlmm 28 GPIO_ACTIVE_HIGH>;
+		};
+		lte_pwrkey {
+			gpio-export,name = "lte_pwrkey";
+			gpio-export,output = <1>;
+			gpios = <&tlmm 29 GPIO_ACTIVE_HIGH>;
+		};
+		lte_usb_boot {
+			gpio-export,name = "lte_usb_boot";
+			gpio-export,output = <0>;
+			gpios = <&tlmm 30 GPIO_ACTIVE_HIGH>;
+		};
+		lte_pwm {
+			gpio-export,name = "lte_pwm";
+			gpio-export,output = <1>;
+			gpios = <&tlmm 31 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	soc {
+		rng@22000 {
```

Comment:
Please use existing labels for these nodes, no reason not to.

Reply:
Can you give an example , or tell stright how this label should look like ?

Reply:
Sure, look at:
https://github.com/openwrt/openwrt/blob/main/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4029-ws-ap391x.dts#L122

You can find all of the labels you can reference in:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/arm/boot/dts/qcom-ipq4019.dtsi?h=v6.1.77#n173

Reply:
Ok , modified  acc.to example . So also other things moved from soc{} section.